lib/MultiModule.js

```diff
@@ -21,12 +20,14 @@ MultiModule.prototype.constructor = MultiModule;
 
 MultiModule.prototype.identifier = function() {
 	return "multi " + this.dependencies.map(function(d) {
-		return path.resolve(this.context, d.request);
-	}.bind(this)).join("!");
+		return d.request;
+	}.bind(this)).join(" ");
```

Comment:
this should use second parameter of `map` or even better you can use an arrow function